src/content/configuration/resolve.md

```diff
@@ -488,6 +489,42 @@ module.exports = {
 };
 ```
 
+### `resolve.importsFields`
+
+`string[]`
```

Comment:
https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types
```suggestion
`[string]`
```